android/assets/jsons/translations/German.properties

```diff
@@ -968,8 +961,7 @@ Downloaded! = Heruntergeladen!
 Could not download mod = Konnte Modifikation nicht herunterladen
  # Requires translation!
 Online query result is incomplete = 
- # Requires translation!
-No description provided = 
+No description provided = Keine Beschreibung mitgeliefert
```

Comment:
Eigentlich sollte man schreiben "Der Autor hielt es nicht für nötig, in seinem mod-repository das 'About' auszufüllen, also vergiß' den Mist hier"... :laughing: Sorry. Ist prima so. Ich wollte ja nur daß da nicht mehr "null" steht...

Reply:
xD